lib/active_publisher/logging.rb

```diff
@@ -0,0 +1,27 @@
+# Taken from https://github.com/mperham/sidekiq/blob/7f882787e53d234042ff18099241403300a47585/lib/sidekiq/logging.rb
+require "time"
+require "logger"
+
+module ActivePublisher
+  module Logging
+    def self.initialize_logger(log_target = STDOUT)
+      oldlogger = defined?(@logger) ? @logger : nil
+      @logger = Logger.new(log_target)
+      @logger.level = Logger::INFO
+      oldlogger.close if oldlogger && !$TESTING # don't want to close testing's STDOUT logging
+      @logger
+    end
+
+    def self.logger
+      defined?(@logger) ? @logger : initialize_logger
+    end
+
+    def self.logger=(log)
+      @logger = (log ? log : Logger.new("/dev/null"))
```

Comment:
I copied this file over from action_subscriber so I don't know a lot of the whys here. I just took a look at action_subscriber and it doesn't look like we are using this setter over there at all. @mmmries, do you know what we'd need to have this setter for or is it safe to take it out?